alf/algorithms/algorithm.py

```diff
@@ -903,6 +903,8 @@ def state_dict(self, destination=None, prefix='', visited=None, **kwargs):
                 (modules, params, algorithms etc) as the key used in the
                 state dictionary.
             visited (set): a set keeping track of the visited objects.
+            kwargs: additional keyword arguments for backward compatibility (
+                e.g., 'keep_vars'). They are not used in newer pytorch versions.
```

Comment:
We can mention `They are not used in newer pytorch versions > 0.4.0.` so that users know right away their code will be compatible.

Reply:
> We can mention `They are not used in newer pytorch versions > 0.4.0.` so that users know right away their code will be compatible.

Updated. But ALF never supports torch<0.4.0. So a user won't even know or worry about this...